ui.frontend/src/main/webpack/components/layout-container/modal.js

```diff
@@ -23,27 +23,33 @@ jQuery(function($) {
         const xfUrl = $(this).data('modal-url');
 
         if (visible || !xfUrl) { return; }
+        const showModalEvt = new Event('wknd-modal-show');
+        const body = document.querySelector('body');
 
         $.get(xfUrl, function (data) {
             const modal = $('<div id="wknd-modal"/>');
             $('body').append(modal.append(data));
             modal.fadeIn(300, function() { visible = true; });
             visible = true;
+            //dispatch event
+            body.dispatchEvent(showModalEvt);
         });
 
         return false;
     }
 
     function hideModal(e) {
         const modal = $('#wknd-modal');
-
+        const body = document.querySelector('body');
+        const hideModalEvt = new Event('wknd-modal-hide');
```

Comment:
What's this event for?

Reply:
its not used...seemed strange to only add an event for Show but not hide...